install.sh

```diff
@@ -345,23 +367,14 @@ elif [ -n "$url_document" ]; then
     cmd="wasmedge --dir .:. \
     --nn-preload default:GGML:AUTO:$chat_model_name \
     --nn-preload embedding:GGML:AUTO:$embedding_model_name \
-    llama-api-server.wasm -p $prompt_type \
+    rag-api-server.wasm -p $prompt_type \
     --model-name $chat_model_stem,$embedding_model_stem \
     --ctx-size $chat_ctx_size,$embedding_ctx_size \
-    --qdrant-url http://127.0.0.1:6333 \
```

Comment:
Where should we set qdrant options for RAG api server?

Reply:
Still by CLI options.

![image](https://github.com/GaiaNet-AI/gaianet-node/assets/4726889/94f92f3c-6691-477c-aaf8-06e57faf750c)